src/ErrorBoundary.js

```diff
@@ -66,4 +68,16 @@ class ErrorBoundary extends Component {
   }
 }
 
+
+export const withErrorBoundary: Function = (
+    Component: Class<React.Component<*>>,
+    FallbackComponent: Class<React.Component<*>>,
```

Comment:
I think `Component` and `FallbackComponent` should both use `ComponentType<any>`, so:

```js
import type { ComponentType } from 'react';

// Further down
Component: ComponentType<any>,
FallbackComponent: ComponentType<any>,
```